### DISCUSSION TITLE
Can't seem to be able to enable Firenvim once `takeover` is set to `never` in the config

### DISCUSSION THREAD

**Discussion by farzadmf** [glacambre/firenvim#1601] [Q&A] [answered]
Hi everyone.

I see the README mentions a way to [manually trigger firenvim](https://github.com/glacambre/firenvim?tab=readme-ov-file#manually-triggering-firenvim). However, when I set `takeover = 'never'` in my config, it seems like Firenvim goes to "permanent disabled mode".

I tried:
- Opening the popup and toggling enable/disable
- Assigning a shortcut to "Toggle Firenvim in the current tab"

But none of them worked.

So, basically my config is like this (using `lazy.nvim`):
```lua
init = function()
  vim.g.firenvim_config = {
    localSettings = {
      ['.*'] = {
        takeover = 'never',
      }
    }
  }
end

```

**Comment by glacambre** [comment]
Does the keyboard shortcut work if you use `takeover = 'once'` instead?

**Comment by farzadmf** [comment, marked as answer]
@glacambre thank you for your reply. Sorry, I missed this one.

Not sure if it's expected or not, but seems like the proper option to use is the one saying `Turn the currently focused element into a neovim iframe`; I've set `shift+alt+q` for that option (along with the above setup), and things are working as expected
